MatrixSDK/VoIP/MXCall.m

```diff
@@ -167,12 +168,15 @@ - (void)handleCallEvent:(MXEvent *)event
                 callStackCall.audioToSpeaker = _isVideoCall;
                 
                 [self setState:MXCallStateWaitLocalMedia reason:nil];
-                
+
+                MXWeakify(self);
                 [callStackCall startCapturingMediaWithVideo:self.isVideoCall success:^{
-                    [callStackCall handleOffer:callInviteEventContent.offer.sdp
+                    MXStrongifyAndReturnIfNil(self);
+
+                    [self->callStackCall handleOffer:self->callInviteEventContent.offer.sdp
                                        success:^{
                                            // Check whether the call has not been ended.
-                                           if (_state != MXCallStateEnded)
+                                           if (self.state != MXCallStateEnded)
```

Comment:
I would expect MXStrongifyAndReturnIfNil(self); at the beginning of the success block